src/cms-content/infoTooltips/index.ts

```diff
@@ -0,0 +1,8 @@
+import { Markdown } from '../utils';
+
+export interface InfoTooltip {
+  title: string;
+  id: string;
+  body: Markdown;
+  cta: Markdown;
+}
```

Comment:
should we import the JSON and export the content here?

Reply:
yep! gonna in the next pr